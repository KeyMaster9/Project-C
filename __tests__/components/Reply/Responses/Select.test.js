import React from 'react';
import renderer from 'react-test-renderer';

beforeEach(() => {
    jest.restoreAllMocks();
});

it('should render a TravelAgent Component', () => {

    const mockValue = '';
    const mockSetValue = () => null;
    const mockOptions = ['first option', 'next option']

    const { default: Select } = require('../../../../src/components/Reply/Responses/Select.jsx');
    const component = renderer.create(<Select value={mockValue} setValue={mockSetValue} options={mockOptions} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

