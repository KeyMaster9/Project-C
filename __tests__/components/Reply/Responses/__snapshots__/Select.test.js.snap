// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a TravelAgent Component 1`] = `
<div
  className="Selection"
>
  <select
    name="response"
    onChange={[Function]}
    value=""
  >
    <option
      value=""
    >
      Select...
    </option>
    <option
      value="first option"
    >
      first option
    </option>
    <option
      value="next option"
    >
      next option
    </option>
  </select>
</div>
`;
